README.md

```diff
@@ -30,50 +29,47 @@ $ pip install meeshkan
 
 Note that `meeshkan` requires **Python 3.6+.**
 
-## Python API
+## Hello world
```

Comment:
I'd suggest turning this into a broader section like `Getting started with Meeshkan` or `Getting up-and-running with Meeshkan` - then I would also add something about the tutorial here too.

So it could be like...
- Flow description
- Tutorial 
- Hello world